### PR TITLE
New species selection menu

### PR DESCRIPTION
**Pull request by FlattestGuitar**
* This respects playtime, but it needs to be further tested.
* This changes config, because playtime requirements are now in it.
* Appreciate this.

# THIS REQUIRES A CONFIG CHANGE ON THE LIVE SERVER

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -398,13 +398,14 @@ JOIN_WITH_MUTANT_RACE
...
 
-## Races that are okay-ish on the station. Safe to enable.
-ROUNDSTART_RACES unathi
-#ROUNDSTART_RACES plasmaman
-#ROUNDSTART_RACES slime
-ROUNDSTART_RACES ethari
-#ROUNDSTART_RACES pod
-ROUNDSTART_RACES ipc
+## Races that are okay-ish on the station. Safe to enable. The number is the minimum hours required to play.
+ROUNDSTART_RACES unathi 10
```
Comment (SamHPurp) [inline config/game_options.txt]: wot

File: config/game_options.txt [hunk 2]
```diff
@@ -398,13 +398,14 @@ JOIN_WITH_MUTANT_RACE
...
+#ROUNDSTART_RACES plasmaman 0
+#ROUNDSTART_RACES slime 10
+ROUNDSTART_RACES ethari 0
```
Comment (SamHPurp) [inline config/game_options.txt]: If you want to timelock either unathi or ethari - you pick ethari, every time.

Either don't timelock either, or do Ethari instead, yo.

**Comment by FlattestGuitar** [comment]
The playtime checks are all wrong. I'll get to it when I have some time tomorrow. No merge for now.

**Comment by Shadeykins** [comment]
You should consider capitalizing the first letter in the species names and justifying the text to the left.

Unsure of how seriously you guys intend to take yourselves/the descriptions, but those could use polish unless you're aiming for a kind of jokey atmosphere in general.

**Comment by SamHPurp** [comment]
Shadey is right on all accounts.

Also, again, Unathi are not going to be locked at start. That is ridiculous.

**Comment by optimumtact** [comment]
hey that's pretty good

**Pull request merged**